elementary/monitor/dbt_project/macros/get_test_rows_sample.sql

```diff
@@ -26,15 +26,5 @@
         {{ return(result_rows) }}
     {% endif %}
 
-    {% if should_query_dwh %}
-      {% set query %}
-        with test_results as (
-          {{ test_query }}
-        )
-        select * from test_results {% if should_limit_sample %} limit {{ results_sample_limit }} {% endif %}
-      {% endset %}
-      {% do return(elementary.agate_to_dicts(elementary.run_query(query))) %}  
-    {% else %}
-      {% do return(elementary.agate_to_dicts([])) %}
-    {% endif %}
+    {% do return(elementary.agate_to_dicts([])) %}
```

Comment:
Why do we use `agate_to_dicts` here instead of `return([])`?

Reply:
No real reason, a changed I did when checked it and forgot it like that.
Changing